Trustroots/trustroots
# Add React and an example component

#### Proposed Changes

* add react just to see how it works/feels

#### Testing Instructions

* run up as normal
* look on profile page
* the "Member since..." and "Online..." boxes on the left are rendered with a react component!

It's just a proof of concept to see how it integrates. That particular component has a load of other stuff too, and maybe not worth switching it over, but yeah, just gives an example. Seems quite nice!
